File: Actividad P2A1/Ejercicio6_Raiz.py
# Ian Alexander Paz Hernandez
# ID: 00603548
# Ingenieria: Tecnologias de la Informacion y Negocios Digitales

import math

num1 = int(input("Ingresa el primer numero: "))
num2 = int(input("Ingresa el segundo numero: "))

cuadrado = math.sqrt((num1 * num2)/2)
cubo = math.cbrt((num1 ** 2 + num2 ** 3) / 4 * 3)

print(f'La raiz cuadrada es: {cuadrado}')
print(f'La raiz cubica es: {cubo}')

File: Actividad P2A1/Ejercicio6_Raiz.cpp
// Ian Alexander Paz Hernandez
// ID: 00603548
// Ingenieria: Tecnologias de la Informacion y Negocios Digitales

#include <iostream>
#include <cmath>

using namespace std;

int main () {
  int num1, num2;
  double cuadrado, cubo;

  cout << "Ingresa el primer numero: " << endl;
  cin >> num1;
  cout << "Ingresa el segundo numero: " << endl;
  cin >> num2;

  cuadrado = sqrt((num1 * num2 /2));
  cubo = cbrt((pow(num1, 2) + pow(num2, 3)) / 4 * 3);

  cout << "La raiz cuadrada es: " << cuadrado << endl;
  cout << "La raiz cubica es: " << cubo << endl;
  
  return 0;
}
